### ISSUE TITLE
Low accuracy on test set

### ISSUE THREAD

**Issue by jingweizhang-xyz** [ImagingLab/ICIAR2018#12]
Hi,

I tried to use the provided pretrained models to predict on test set provided on BACH website. I added a csv writer, replace "volatile" with "with torch.no_grad()" to fix the outofmemory error. I uploaded the predicted labels to the challenge website and got the accuracy on this item being only 0.66. Could you please provide some thoughts on what the problem is?

Thank you!
